### ISSUE TITLE
Format/style changes

### ISSUE THREAD

**Issue by seani** [OneToOneService/iService-Forms#84]
- Make all indentation 2 spaces.
- Only use double quotes where absolutely necessary (such as in HTML elements).
- Because we don't minimize, remove comments.
- Remove all unused CSS.
- Format CSS like the static iService CSS files, with each directive on a single line.

**Comment by ashrafsabrym** [comment]
Fixed in PR #86

**Comment by ashrafsabrym** [comment]
Cleaning the CSS was the toughest part. There were hundreds of unused rules, so I couldn't do it manually. I wrote the following script to clean the unused rules:

``` javascript
let styleElement = document.getElementById("st"),
    sheet = styleElement.sheet,
    cssRules = sheet.cssRules,
    countDeleted = 0,
    output = document.createElement("textarea"),
    styleText = styleElement.textContent,
    selectorsBag = new Set();

function traverseRule(rl)
{
    /// <param name="rl" type="CSSStyleRule"></param>
    if(rl.type == CSSRule.STYLE_RULE)
    {
        let selectorText = rl.selectorText;

        if(selectorText.indexOf(":after") < 0 &&
           selectorText.indexOf(":before") < 0 &&
          !selectorsBag.has(selectorText) &&
          !document.querySelector(selectorText))
        {
            let textLength = styleText.length;

            styleText = styleText.replace(new RegExp("((?:^[\\s\\r\\n]*)|(?:\\}\\s*))" + selectorText.replace(/[\\^$*+?.()|[\]{}]/g, '\\$&') + "[\\s\\r\\n]*\\{[^\\{\\}]*\\}", "gim"), "$1");

            if(textLength == styleText.length)
                console.log("Failed to replace: " + selectorText);
            else
            {
                selectorsBag.add(selectorText);

                countDeleted++;
            }
        }
    }
    else if(rl.cssRules)
    {
        for(let i = rl.cssRules.length; i--;)
            traverseRule(rl.cssRules[i]);
    }
}

function cleanCss()
{
    for(let i = cssRules.length; i--;)
        traverseRule(cssRules[i]);

    console.log(`Deleted ${countDeleted} rules`);

    output.value = styleText;

    document.body.appendChild(output);
}
```

I post it just in case anyone needed to do this in another form.
